lua/entities/gmod_wire_expression2/core/debug.lua

```diff
@@ -197,14 +197,11 @@ end
 
 --[[******************************************************************************]]
 
-local valid_print_types = {
-	HUD_PRINTNOTIFY = HUD_PRINTNOTIFY,
```

Comment:
can't you just fix the table setting the keys as value? eg: "HUD_PRINTNOTIFY" = HUD_PRINTNOTIFY